content/en/docs/examples/virtual-machines/autoregistration/index.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Automated Workload Entry Creation [experimental]
+description: Learn how to use the experimental automated VM registration feature.
+  Istio mesh.
+weight: 20
```

Comment:
This value controls where in the list of documents under the same folder this one should be. The other docs are currently, 20, 30, and 60. If you want it to be first, set `weight: 10` or last `weight: 70`
```suggestion
weight: 70
```